### ISSUE TITLE
nonce sleep?

### ISSUE THREAD

**Issue by perlancar** [ngeong/btcid#1]
hi :)

saya perhatiin di kodenya nonce pake sleep(1), apakah gak mengganggu kalo harus sering call API? misalnya dalam kode bot yang harus ngecek last trades + depth, lalu list open orders, lalu open order? bisa jadi berdetik2 delaynya.

saya perhatiin juga, si app android bitcoin.co.id menggunakan nonce yang besar nilainya (>32 bit) dan diizinkan. mungkin bisa dipertimbangkan nonce-nya menggunakan unix time dalam milidetik, seperti yg saya pakai di perl wrapper https://github.com/perlancar/perl-Finance-BTCIndo

**Comment by ngeong** [comment]
Iya, saya juga sudah ubah jadi milidetik pakai unix time. Untuk penggunaan bot awalnya emang jadi delay lama.
